### PR TITLE
[ENH] Adding LITETime for Deep Learning classifiers

### PR DESCRIPTION
**Pull request by hadifawaz1999**
Adding LITE and LITETime from [LITE: Light Inception with boosTing tEchniques for Time Series Classification](https://ieeexplore.ieee.org/abstract/document/10302569)

Same setup as the rest.

### CODE REVIEW

**Review by TonyBagnall — APPROVED** [review]
LGTM

File: aeon/classification/deep_learning/lite_time.py [hunk 1]
```diff
@@ -0,0 +1,559 @@
...
+        list of tf.keras.callbacks.Callback objects.
+    file_path : str, default = "./"
+        file_path when saving model_Checkpoint callback
+    save_best_model : bool, default = False
+        Whether or not to save the best model, if the
+        modelcheckpoint callback is used by default,
+        this condition, if True, will prevent the
+        automatic deletion of the best saved model from
+        file and the user can choose the file name
+    save_last_model     : bool, default = False
```
Comment (TonyBagnall) [inline aeon/classification/deep_learning/lite_time.py:60]: soz to be picky, but could you sort the spacing out here?

File: aeon/classification/deep_learning/lite_time.py [hunk 2]
```diff
@@ -0,0 +1,559 @@
...
+        save_best_model is set to False, this parameter
+        is discarded
+    last_file_name      : str, default = "last_model"
+        The name of the file of the last model, if
+        save_last_model is set to False, this parameter
+        is discarded
+    random_state        : int, default = 0
+        seed to any needed random actions.
+    verbose             : boolean, default = False
+        whether to output extra information
```
Comment (TonyBagnall) [inline aeon/classification/deep_learning/lite_time.py:328]: and here

File: aeon/networks/lite.py [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+        The activation function used in each lite layer, if not a list,
+        the same is used in all lite layers.
+
+    Adapted from the implementation from Ismail-Fawaz et. al
+
+    https://github.com/MSD-IRIMAS/LITE
+
+    Network originally defined in:
+
+    @inproceedings{ismail-fawaz2023lite,
```
Comment (TonyBagnall) [inline aeon/networks/lite.py:32]: could you put this as a reference rather than bibtex, in format of others?

**Review by TonyBagnall — CHANGES_REQUESTED** [review]

File: aeon/classification/deep_learning/lite_time.py (hunk 1 above)
Comment (hadifawaz1999) [reply aeon/classification/deep_learning/lite_time.py]: thanks forgot about that, fixed now

File: aeon/classification/deep_learning/lite_time.py (hunk 2 above)
Comment (hadifawaz1999) [reply aeon/classification/deep_learning/lite_time.py:328]: fixed

File: aeon/networks/lite.py (hunk 3 above)
Comment (hadifawaz1999) [reply aeon/networks/lite.py]: fixed

**Review by TonyBagnall — APPROVED** [review]

**Pull request merged**